iphone/Classes/TiContactsGroup.m

```diff
@@ -283,7 +283,10 @@ -(void)add:(id)arg
 					   subreason:nil
 						location:CODELOCATION];
 		};
+		//Ignore static analylzer warning here
```

Comment:
Same as above, please describe why (because of the release-cycle later, yeah, but only we both know that :-)

Also please try to leave a space in the comments and start capitalized for an improved readability.